config.json

```diff
@@ -0,0 +1,83 @@
+{
+  "settings": {
+    "Project": {
+      "description": "Project where the commands are going to be applied.",
+      "valueType": "passThrough",
+      "values": [],
+      "defaultValue": ""
+    },
+    "CleanArtifacts": {
+      "description": "MsBuild target that deletes the artifacts output directory.",
+      "valueType": "target",
+      "values": [],
+      "defaultValue": ""
+    },
+    "CleanPackages": {
+      "description": "MsBuild target that deletes the repo-local nuget package directory.",
```

Comment:
There are several of these. artifacts/packages and artifacts/test/packages and .nuget...